### ISSUE TITLE
Error occurred using the latest docker image

### ISSUE THREAD

**Issue by paprikachan** [polyactis/Accurity#1]
Hi, 

I used the latest version of docker image to run Accuracy, while following error occured:

```
/usr/local/Accurity/GADA: error while loading shared libraries: libhdf5_cpp.so.11: cannot open shared object file: No such file or directory
```

Thanks, Lingxi

**Comment by polyactis** [comment]
We forgot to add one library into it. While we are updating the docker, you
can install it inside the docker by typing this line:

apt install libhdf5-cpp-11

**Comment by paprikachan** [comment]
Yeah, I solved it by  apt-get install libhdf5-dev. Wait for your new version; )

Thanks a lot.

**Comment by polyactis** [comment]
It's updated.

Let me know if you encounter any issues

**Comment by paprikachan** [comment]
@polyactis 

Another error found in plot stage:

```
[plot_model_select] ImportError: No module named h5py
```

Thanks.

**Comment by polyactis** [comment]
it's another missing library due to development version. we will add a
docker test in our software testing pipeline to catch this kind of error.

run 'apt install python-h5py' to fetch the library.

If you have run till this far, you should have got all output, minus some
plots.  You can rerun the same  main.py commandline with "-s 6" in the end
to get all plots.

The CNV plot is * plot.cnv.png*

**Comment by paprikachan** [comment]
I rerun the program with '-s 6', while the file `model_selection_log/model_selection.h5` is missing. 
Which step generates this file?

**Comment by polyactis** [comment]
This .h5 output function is written by Xinping (cc-ed). he'll reply after
back from a meeting.

Try "-s 5". if not good, wait for his email

**Comment by polyactis** [comment]
Step 5 will generate the file 'model_selection.h5'. But firstly, you should check whether the directory 'model_selection' exists in your output directory. In general, step 0 will create that directory in your output directory automatically, and you can create that directory by hand if you can't find that.

------------------------------------------------------------------
Sender:Yu Huang <polyactis@gmail.com>
Sent At:2019 Jan. 16 (Wed.) 14:10
Recipient:polyactis/Accurity <reply@reply.github.com>
Cc:范新平 <fanxp@yfish.onaliyun.com>
Subject:Re: [polyactis/Accurity] Error occurred using the latest docker image (#1)

This .h5 output function is written by Xinping (cc-ed). he'll reply after back from a meeting.

Try "-s 5". if not good, wait for his email




On Wed, Jan 16, 2019 at 2:08 PM Lingxi Chen <notifications@github.com> wrote:

I rerun the program with '-s 6', while the file model_selection_log/model_selection.h5 is missing.
 Which step generates this file?
